### ISSUE TITLE
Optionally write rendered html to .put directory 

### ISSUE THREAD

**Issue by vvisigoth** [tadad/blog-ui#6]
Writing out the blog to a unix directory would be useful if you ever wanted to put blog behind nginx.

**Comment by tadad** [comment]
good idea - I think I'm going to to implement so that you can download all of your html/md from the browser. Mostly because I don't think most people want to screw around in unix

**Comment by tadad** [comment]
now you can export markdown easily by just add `.md` to the URL. so if your post is at `http://sampel-palent.urbit.org/blog/post-1`, you can get the markdown at `http://sampel-palent.urbit.org/blog/post-1.md`
